bundles/org.openhab.binding.paradoxalarm/src/main/resources/ESH-INF/thing/panel.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="paradoxalarm"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Sample Thing Type -->
+	<thing-type id="panel">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="ip150" />
+		</supported-bridge-type-refs>
+
+		<label>Paradox Panel</label>
+		<description>Paradox panel various information</description>
+
+ 		<channels>
+			<channel id="state" typeId="panelState" />
+			<!-- <channel id="serialNumber" typeId="serialNumber" />

```

Comment:
Can you remove those

Reply:
Hm... those were removed upon request by Hilbrand... it seems I've reverted some of the last changes. I need to get back to Hilbrands latest requests and verify everything is fixed again. :(